src/Agent.Worker/WorkerUtilties.cs

```diff
@@ -78,9 +78,11 @@ public static Pipelines.AgentJobRequestMessage ScrubPiiData(Pipelines.AgentJobRe
             scrubbedJobResources.SecureFiles.AddRange(message.Resources.SecureFiles);
 
             // Reconstitute a new agent job request message from the scrubbed parts
-            return new Pipelines.AgentJobRequestMessage(
+            var requestMessage = new Pipelines.AgentJobRequestMessage(
```

Comment:
i will fix this, i will let you know when to rebase.

Reply:
alrighty!

Reply:
try rebase master.